backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/deliveryplanning/DeliveryPlanningCreateRequest.java

```diff
@@ -119,6 +119,12 @@
 	@NonNull
 	Quantity actualDeliveredQty;
 
+	@NonNull Quantity plannedLoadedQty;
```

Comment:
i guess this is the pendant to `actualLoadQty`, right? if so, please pick either `load`or `loaded` 🤓